setup.py

```diff
@@ -40,6 +40,7 @@
         'flask-cache>=0.13.1',
         'flask-login>=0.2.11',
         'flower>=0.7.3',
+        'jaydebeapi>=0.2.0',
```

Comment:
Can you switch to using the `extra_requires` to make it optional? You can call it `jdbc`. What this does is that if you want to install this extra, you have to write something like `pip install airflow[jdbc]`